src/Okta.Idx.Sdk/IdxClientExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// <copyright file="IdxClientExtensions.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Okta.Idx.Sdk
+{
+    internal static class IdxClientExtensions
```

Comment:
Should we just move these methods on the IdxClient instead of having a separate file with extensions?

Reply:
I'd prefer to have separate classes in separate files. And we can change it to be public at some point, now it's all like a first approach.

Reply:
There's already an extensions folder I created a few weeks ago. Let's move this file into that folder.